apps/wallet-ui/biome.json

```diff
@@ -7,6 +7,6 @@
     }
   },
   "files": {
-    "ignore": []
+    "ignore": ["./src/components/ui/**/*"]
```

Comment:
why?

Reply:
The shad components do funky things biome doesn't like